test/CMakeLists.txt

```diff
@@ -58,3 +58,10 @@ endif()
 if(PERFORMANCE_TESTS)
     add_subdirectory(performance)
 endif()
+
+###############################################################################
+# System tests
+###############################################################################
+if(PERFORMANCE_TESTS)
```

Comment:
It seems we missed something here on the revision of the original PR. I think this should be a different option `SYSTEM_TESTS`

Reply:
```suggestion
if(SYSTEM_TESTS)
```

Reply:
And also adding the option on line 19